test/parallel/test-fs-read-type.js

```diff
@@ -76,7 +76,7 @@ assert.throws(() => {
            'It must be >= 0. Received -1'
 });
 
-[true, () => {}, {}, ''].forEach((value) => {
+[true, () => {}, {}, '', null, undefined].forEach((value) => {
```

Comment:
```suggestion
[true, () => {}, {}, '', null].forEach((value) => {
```
`undefined` is swapped to `-1`, hence the missing exception.

Reply:
@aduh95 Thanks for fixing this :)